cvat-ui/src/utils/image-processing.tsx

```diff
@@ -11,6 +11,7 @@ export interface ImageProcessing {
 
     processImage: (src: ImageData, frameNumber: number) => ImageData;
     configure: (options: object) => void;
+    serialize: () => any;
```

Comment:
the default method for this is `toJSON`
Thus the class will be supported by JSON.stringify

Reply:
Renamed & added type